sdks/go/pkg/beam/core/runtime/exec/combine_test.go

```diff
@@ -60,9 +170,30 @@ func TestCombine(t *testing.T) {
 	if err := p.Down(context.Background()); err != nil {
 		t.Fatalf("down failed: %v", err)
 	}
+}
 
-	expected := makeKV(42, 21)
-	if !equalList(out.Elements, expected) {
-		t.Errorf("pardo(sumFn) = %v, want %v", extractKeyedValues(out.Elements...), extractKeyedValues(expected...))
+func mergeFn(a, b int) int {
+	return a + b
+}
+
+type MyCombine struct{}
```

Comment:
Wouldn't hurt to add a comment indicating that int is the input type and int64 is the accumulator and output type. The test is an authoritative piece of code demonstrating the contract. That comment will help people write their own combiners more easily, I believe.